content/05.limitations.md

```diff
@@ -1,14 +1,12 @@
 ## Limitations of Study
 
-This study has some potential limitations.
-The brain tumor samples were collected over decades and RNA samples were prepared using more than one type of library method (stranded or poly-A, **Figure {@fig:S7}A**) by multiple sequencing centers.
-While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and possible sequencing center batch effect (**Figure {@fig:S7}C**), we also found a very unbalanced distribution of cancer groups by library preparation type (**Figure {@fig:S7}A**).
-We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences among groups [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 
-Instead, we analyzed only stranded RNA-Seq expression data, which comprised the vast majority of the PBTA cohort, for transcriptomic analyses presented in **Figure {@fig:4}** and **Figure {@fig:5}** to circumvent the batch differences.
-As batch correction strategy is highly dependent on an analyst's goals [@doi:10.1016/j.tibtech.2017.02.012], we made expression matrices available by library type in the OpenPBTA data release for other researchers to employ based on their given analysis needs. 
+Notably, PBTA brain tumor samples were collected over decades, and RNA samples were prepared using two distinct library preparations (stranded or poly-A, **Figure {@fig:S7}A**) by multiple sequencing centers.
+While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and a possible sequencing center batch effect (**Figure {@fig:S7}C**), cancer groups are highly unbalanced across library preparations (**Figure {@fig:S7}A**).
+We did not perform batch correction because removing batch effects across unbalanced groups may induce false differences among groups [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012].
+Instead, we circumvent batch effects by analyzing only stranded RNA-Seq expression data, which comprises the vast majority of the PBTA cohort, for transcriptomic analyses presented in **Figure {@fig:4}** and **Figure {@fig:5}** .
```

Comment:
```suggestion
Instead, we circumvent batch effects by grouping only stranded RNA-Seq expression data, which comprises the vast majority of the PBTA cohort, for transcriptomic analyses presented in **Figure {@fig:4}** and **Figure {@fig:5}** .
```
reword since we did analyze both (fusions, tp53, etc) - i know it was my original word, too!